From 680a5869845ae2503ee629a2dc4b6c6f790b6bc4 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Fri, 23 May 2025 01:38:31 -0700
Subject: [PATCH 1/2] remove dep nvidia nccl cu12

---
 python-package/pyproject.toml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/python-package/pyproject.toml b/python-package/pyproject.toml
index 5a3b897..ced6a25 100644
--- a/python-package/pyproject.toml
+++ b/python-package/pyproject.toml
@@ -30,8 +30,7 @@ classifiers = [
 ]
 dependencies = [
     "numpy",
-    "scipy",
-    "nvidia-nccl-cu12 ; platform_system == 'Linux' and platform_machine != 'aarch64'",
+    "scipy"
 ]
 
 [project.urls]
-- 
2.39.3 (Apple Git-146)

