From fd00b981e0cec686f7360de63604cb09d93b0eb5 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Wed, 25 Feb 2026 06:34:27 -0800
Subject: [PATCH 2/2] remove hatch custom build hook

---
 python-package/pyproject.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/python-package/pyproject.toml b/python-package/pyproject.toml
index 0513437..8da37ea 100644
--- a/python-package/pyproject.toml
+++ b/python-package/pyproject.toml
@@ -47,8 +47,6 @@ dask = ["dask", "pandas", "distributed"]
 plotting = ["graphviz", "matplotlib"]
 pyspark = ["pyspark>=3.4", "scikit-learn", "cloudpickle"]
 
-[tool.hatch.build.targets.wheel.hooks.custom]
-
 [tool.isort]
 profile = "black"
 
-- 
2.39.3 (Apple Git-146)

