From 57e01b19e27caddc0788d5675f7f176aacbf514f Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Wed, 3 Dec 2025 10:36:31 +0100
Subject: [PATCH 2/2] remove hatch custom build hook

---
 python-package/pyproject.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/python-package/pyproject.toml b/python-package/pyproject.toml
index e80da99..d09ebf9 100644
--- a/python-package/pyproject.toml
+++ b/python-package/pyproject.toml
@@ -46,8 +46,6 @@ dask = ["dask", "pandas", "distributed"]
 plotting = ["graphviz", "matplotlib"]
 pyspark = ["pyspark>=3.4", "scikit-learn", "cloudpickle"]
 
-[tool.hatch.build.targets.wheel.hooks.custom]
-
 [tool.isort]
 profile = "black"
 
-- 
2.39.5 (Apple Git-154)