From 53fe1507f46fef7c3b1efbda8588bb23c9788291 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Fri, 23 May 2025 01:40:30 -0700
Subject: [PATCH 2/2] remove hatch custom build hook

---
 python-package/pyproject.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/python-package/pyproject.toml b/python-package/pyproject.toml
index ced6a25..877a261 100644
--- a/python-package/pyproject.toml
+++ b/python-package/pyproject.toml
@@ -45,8 +45,6 @@ datatable = ["datatable"]
 plotting = ["graphviz", "matplotlib"]
 pyspark = ["pyspark", "scikit-learn", "cloudpickle"]
 
-[tool.hatch.build.targets.wheel.hooks.custom]
-
 [tool.isort]
 profile = "black"
 
-- 
2.39.3 (Apple Git-146)

